docs/features/toolbar.md

```diff
@@ -73,7 +72,11 @@ toolbar: {
 
  * **`removeItems`** &ndash; An array of toolbar item names. With this setting you can modify the default toolbar configuration without the need of defining the entire list (you can specify a couple of buttons that you want to remove instead of specifying all the buttons you want to keep). If, after removing an item, toolbar will have two or more consecutive separators (`'|'`), the duplicates will be removed automatically.
 
- * **`viewportTopOffset`** &ndash; The offset (in pixels) from the top of the viewport used when positioning a sticky toolbar. Useful when a page with which the editor is being integrated has some other sticky or fixed elements (e.g. the top menu). Thanks to setting the toolbar offset, the toolbar will not be positioned underneath or above the page's UI.
+ * **`viewportTopOffset` (deprecated)** &ndash; The offset (in pixels) from the top of the viewport used when positioning a sticky toolbar. Useful when a page with which the editor is being integrated has some other sticky or fixed elements (e.g. the top menu). Thanks to setting the toolbar offset, the toolbar will not be positioned underneath or above the page's UI.
```

Comment:
I'm afraid this bit needs to be rethought. Most of the people who will read this guide ever will not know that this option ever existed. So, it makes sense to only briefly mentioned it that it wasthere or not even mention at all.

Instead, we need to think on how to add a section about `config.ui.viewportSth`. It might be a new section (but it's unclear to me how it could be titled... "Configuring the viewport dimensions"? "Limiting the viewport area"?) or we can just mention this property here like it was part of toolbar configuration. No one said that toolbar config cannot be outside `config.toobar` too. The third option will be to use `<info-box>` to detach this information a bit from its surrounding.

tl;dr: When writing docs (not migration guides) focus on what's available currently.

Reply:
Removed